ReplCommand.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2020 Grakn Labs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package grakn.console;
+
+import grakn.client.Grakn;
+import org.jline.reader.EndOfFileException;
+import org.jline.reader.LineReader;
+import org.jline.reader.UserInterruptException;
+
+public abstract class ReplCommand {
+    static class Exit extends ReplCommand {}
```

Comment:
So i'm actually thinking that we can put the help menu in here? and name of each command?
Rather than hard-coding the "help menu" at the repl level, we can assemble the help menu from here, for instance:
```
HELP_MENU = ReplCommand.Clear.name + ":" + ReplCommand.Clear.description
```
...
what do you think? This way we only have to maintain this file over time with new features

Reply:
can also make a static function that formats it for you somehow...?

Reply:
Emm, this would be nice, but see the below comment, there's no one-to-one mapping between keywords and commands.

Also the help menu we want to show contains template variable, like `database create <db>`.